### PR TITLE
[bump] client: 2.0.1 => 2.0.2 (patch)

### PR DESCRIPTION
**Pull request by daesunp**
## Description

Version bump from 2.0.1 to 2.0.2

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -28252,7 +28252,7 @@ packages:
     resolution: {integrity: sha512-m7Lllj9n/S6sOkCkRftpM7L24uvmfXQFedlW/4hENcuJH1HHm9u5EgxZb9uVjQSCGrbBWBkOGgcTxNg36r6ywA==}
     dependencies:
       fast-glob: 3.3.2
-      minimatch: 9.0.3
+      minimatch: 9.0.4
```
Comment (yann-achard-MS) [inline pnpm-lock.yaml:28255]: The changes in this file seem unrelated to version bump.

Comment (CraigMacomber) [reply pnpm-lock.yaml:28255]: this change is deduplicating 9.0.3 with 9.0.4 (which we already had a copy of included). No idea why this would happen in this change, but seems good and is likely related to the resolution step that is caused during pnpm install due to having file dependencies (which I think just got fixed on main).

Regardless I don't know whats up with the `requiresBuild: true` lines. Liley best to just revert the changes to this file.

Comment (daesunp) [reply pnpm-lock.yaml]: reverted changes from pnpm-lock.yaml file

**Review by yann-achard-MS — APPROVED** [review]

**Review by CraigMacomber — APPROVED** [review]
